src/IntervalOptimisation.jl

```diff
@@ -1,19 +1,20 @@
-
-
 module IntervalOptimisation
 
 export minimise, maximise,
        minimize, maximize
 
+import Base: getindex, length, push!, isempty,
```

Comment:
I still think it would be better to have `SortedVector` and `HeapVector` as modules.

Reply:
May be If you saying so, But as you know I tried with that approach too but it making code unnecessarily complicated and hard to understand for any new person.

Reply:
Anyways, I have pretty much good confidence on this approach too, It wont fail in any case.